lib/physiqual/data_aggregator.rb

```diff
@@ -24,6 +24,7 @@ def heart_rate(from, to)
     def sleep(from, to)
       result = retrieve_data_of_all_services { |service| service.sleep(from, to) }
       run_function(result) do |sleep_data, data_entry|
+        # TODO: Mist hier een .flatten?
```

Comment:
ja dat dacht ik ook, viel me ook al op dat we het bij alle anderen wel doen, maar hier niet. Sleep werkte niet goed in de tests, dus ik kon dit verder nog niet testen (door al die calls naar Fitbit.) Ik zal er nog even naar kijken.